ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+### Prerequisites
+
+We thank you for reporting a bug or feature request and helping us to make the SeqAn library better.
+We would like you to check the following points before continuing with the submission as often
+some items might have been solved already. 
+
+* [ ] Are you running the latest version?
+* [ ] Did you check the debugging guide?
```

Comment:
Add a link.

Reply:
We first need some 😬 
Not sure if we will have one. I would rather take this point out. It seems to me that the complexity behind this might scary people off.

Reply:
I agree. I think users should not be required to debug seqan.